Framework/Kernel/test/EnumeratedStringTest.h

```diff
@@ -7,10 +7,16 @@
 #pragma once
 
 #include "MantidKernel/EnumeratedString.h"
+#include "MantidKernel/Logger.h"
 #include <cxxtest/TestSuite.h>
 
 using namespace Mantid::Kernel;
 
+namespace {
```

Comment:
Are these calls to the logger needed?

Reply:
Yes, this is to help the next programmer with debugging these tests.  These log notices help identify the exact ctest class method(s) that fail.